core/test/runner-test.js

```diff
@@ -283,25 +327,9 @@ describe('Runner', () => {
 
     return runGatherAndAudit(createGatherFn(url),
         {resolvedConfig, driverMock, computedCache: new Map()}).then(_ => {
-      expect(gatherRunnerRunSpy).toHaveBeenCalled();
-      assert.ok(typeof resolvedConfig.passes[0].gatherers[0] === 'object');
-    });
-  });
-
-  it('rejects when given neither passes nor artifacts', async () => {
```

Comment:
The FR runner methods are not responsible for checking this

Reply:
`navigation-runner.js` (sort of)

It rejects when there are no "config navigations" but there should always be 1 even if the `artifacts` array on the config doesn't exist or is empty.